EventFilter/SiPixelRawToDigi/src/PixelUnpackingRegions.cc

```diff
@@ -77,13 +77,11 @@ void PixelUnpackingRegions::initialize(const edm::EventSetup& es) {
   // initialize cabling map or update it if necessary
   // and re-cache modules information
   if (watcherSiPixelFedCablingMap_.check(es)) {
-    edm::ESTransientHandle<SiPixelFedCablingMap> cablingMap;
-    es.get<SiPixelFedCablingMapRcd>().get(cablingMap);
+    edm::ESHandle<SiPixelFedCablingMap> cablingMap = es.getHandle(cablingMapToken_);
```

Comment:
NOTE: The previous use of `ESTransientHandle` is inappropriate here as the data product can not be allowed to be deleted before the end of the IOV as this module holds onto parts of the data product.